src/Stryker.Core/Stryker.Core.UnitTest/Mutators/InitializerMutatorTests.cs

```diff
@@ -34,6 +34,28 @@ public void ShouldRemoveValuesFromArrayInitializer()
             replacement.Expressions.ShouldBeEmpty();
         }
 
+        [Fact]
+        public void ShouldRemoveValuesFromObjectInitializer()
```

Comment:
could you add a test for dictionary initializer?

Reply:
Also: I prefer to use `SyntaxFactory.ParseExpression("code here")` in unit tests as it is more readable than using the SyntaxFactory to create your nodes.

Reply:
The dictionary test apparently already existed in the existing mutator.
Switched to using ParseExpression 👍